src/index.js

```diff
@@ -98,16 +98,22 @@ class FlipMotion extends Component<void, Props, State> {
     }
   }
   render() {
-    const { style, childStyle } = this.props
+    const style = this.props.style
+    const childStyle = this.props.childStyle
+    const Element = this.props.element || "div"
+    const ChildElement = this.props.childElement || "div"
```

Comment:
`element` & `childElement` should be called `component` & `childComponent` as that's what they are semantically